scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -5,18 +5,19 @@ def printBigHeader( author = 'Jungle Cat Software', package = 'AWS-Toolkit', fun
   firstLineLen = len( firstLine )
   secondLine = function
   secondLineLen = len( secondLine )
-
+  
   longestLineLen = firstLineLen if firstLineLen >= secondLineLen else secondLineLen
   linePadLen = longestLineLen + 2
   headerLineLen = linePadLen + 2
 
-  print( ( '{0:*^' + str( headerLineLen ) + '}' ).format( '' ) )
-  print( ( '*{0: ^' + str( linePadLen ) + '}*' ).format( firstLine ) )
-  print( ( '*{0: ^' + str( linePadLen ) + '}*' ).format( secondLine ) )
-  print( ( '{0:*^' + str( headerLineLen ) + '}' ).format( '' ) )
+  print('*' * headerLineLen)
+  formatStr = '*{{0: ^{}}}*'.format(linePadLen)
+  print(formatStr.format(firstLine))
+  print(formatStr.format(secondLine))
+  print('*' * headerLineLen)
 
 def printSmallHeader( function = 'Output Formatter' ):
-  print( '\n==' + function + '==' )
+  print('\n=={}=='.format(function))
```

Comment:
With strings this small, it doesn't really make that much difference, but a good habit to get into is not to use "string" + "string" in Python.  It's expensive, because for each +, a new string is allocated in memory.  Plus, the point you make about consistency is valid.  :+1: